### PR TITLE
Misc fixes for react native

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: shared/react/react.force.smartstore.js [hunk 1]
```diff
@@ -36,11 +36,11 @@ var exec = function(successCB, errorCB, methodName, args) {
     console.log(func + " called: " + JSON.stringify(args));
     SFSmartStoreReactBridge[methodName](args, function(error, result) {
         if (error) {
-            storeConsole.debug(func + " failed: " + error);
+            console.log.log(func + " failed: " + JSON.stringify(error));
```
Comment (bhariharan) [inline shared/react/react.force.smartstore.js]: ```.log.log```?

File: shared/react/react.force.net.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ var exec = function(path, successCB, errorCB, method, payload, headerParams) {
     console.log(func + " called with path:" + path + " and method:" + method);
     SFNetReactBridge.sendRequest(args, function(error, result) {
         if (error) {
-            console.log(func + " failed: " + error);
+            console.log.log(func + " failed: " + JSON.stringify(error));
```
Comment (bhariharan) [inline shared/react/react.force.net.js]: Is it ```.log.log```?

**Comment by wmathurin** [comment]
@bhariharan Typo fixed https://github.com/wmathurin/SalesforceMobileSDK-iOS/commit/5a229263187176e3a446dcca83e2c6f2c8aee5c2

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**